### ISSUE TITLE
Using it with Xlabs IPAdapter & Controlnet

### ISSUE THREAD

**Issue by ardha27** [balazik/ComfyUI-PuLID-Flux#24]
Hi, are we able to use it together with IPAdapter & Controlnet from Xlabs?

**Comment by seasnakes** [comment]
yes,I try and they can be used together

**Comment by ardha27** [comment]
> yes,I try and they can be used together
May i get your workflow, i try it but cant get a good result

**Comment by seasnakes** [comment]
> > yes,I try and they can be used together
> > May i get your workflow, i try it but cant get a good result
[workflow_api (17).json](https://github.com/user-attachments/files/17306790/workflow_api.17.json)
![workflow (36)](https://github.com/user-attachments/assets/e1406880-a6a8-4e48-90eb-12a8c81a9bb0)

**Comment by seasnakes** [comment]
> > yes,I try and they can be used together
> > May i get your workflow, i try it but cant get a good result

the author of the repo is still working on the part of attention mask ,so multiperson pulid does't work properly(only one face effects)

**Comment by balazik** [comment]
Yes, I kept the ControlNet hook untouched so it can work with it. 

I have also a very early working prototype of the masking feature ready, but it need's some more time to be published. (need make more overrides so I can get to the original latent size => and I want to use the ModelPatcher patch / unpatch feature so it can be detached from the graph when needed).
